availability-monitoring-oschecks.txt

```diff
@@ -0,0 +1,263 @@
+OS - OPSTOOLS - OSCHECKS - MONITORING
+=====================================
+:Author:   Juan Badia Payno
+:Email:    jbadiapa@redhat.com
+:Date:     2016-Agust-05
+:Revision: v1
+
+== Installation
+
+[source,bash]
+yum install osops-tools-monitoring-oschecks 
+
+== Usage
+
+.oschecks-check_amqp
+Checks whether the program is running and is connected to the AMQP 
+
+[source,bash]
+oschecks-check_amqp sensu-client
+
+.oschecks-check_ceilometer_api
+
+Checks the amount os meters beeing used
+[source,bash]
+oschecks-check_ceilometer_api --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
+//.oschecks-check_glance_api
+
+//[source,bash]
+//oschecks-check_glance_api --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
+.oschecks-check_cinder_api
+  
+Check the list of the quotas by a user
+[source,bash]
+oschecks-check_cinder_api --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
+.oschecks-check_keystone_api
+
+Check how long it takes to get the token. There is a warning alarm if it takes longer than 10 seconds
+
+[source,bash]
+oschecks-check_keystone_api --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
+.oschecks-check_nova_api
+
+Get the amount of flavors
+[source,bash]
+oschecks-check_nova_api --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+   
+//.oschecks-check_neutron_floating_ip
+
+//[source,bash]
+//oschecks-check_neutron_floating_ip --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
+
+.oschecks-check_cinder_volume
+
+It checks the existing of ta already created volume(for testing purpose), and also the creation/delete of a volume
+
+. Check the existence of the volume
+. Create a volume
+. Check the status of the volume
+. Delete the volume
+. Ensure the volume was deleted
+
+[source,bash]
+oschecks-check_cinder_volume  --username USER --password PASSWORD --auth_url AUTH_URL --tenant TENANT
+
+//.oschecks-check_ceph_health        
+  
+//[source,bash]
+//oschecks-check_ceph_health --os-username USER --os-password PASSWORD --os-tenant-name TENANT  --os-auth-url AUTH_URL
+
```

Comment:
I don't think we need OpenStack credentials when testing Ceph. Are you sure about this?